### ISSUE TITLE
Cannot Relock VMware Workstation 17.0

### ISSUE THREAD

**Issue by hindsa-nabeel** [DrDonk/unlocker#69]
When I try to relock VMware Workstation 17.0 by executing relock.exe, I get the below error!

Relocker 4.2.3 for VMware Workstation/Player
============================================
© 2014-2022 David Parsons

VMware is installed at:  C:\Program Files (x86)\VMware\VMware Workstation\
VMware version:  20800274

Stopping VMware services and tasks...
Stopping service  VMUSBArbService
Stopping service  VMAuthdService
Stopping service  VmwareAutostartService
Stopping task  vmware-tray.exe
Uninstalling unlock patches
Aborting uninstall as backup folder does not exist!

Starting VMware services and tasks...
Starting service  VMAuthdService
Starting service  VMUSBArbService
Starting service  VmwareAutostartService
Starting task  C:\Program Files (x86)\VMware\VMware Workstation\vmware-tray.exe

Press Enter key to continue...

**Comment by DrDonk** [comment]
Can you get me the unlocker folder contents please? And specifically the backup folder.

**Comment by hindsa-nabeel** [comment]
I got it, the name of the folder inside the backup is the full version of VMware Workstation that was unlocked. I removed Version 17.0 and reinstalled Version 16.2.4 and performed the relock; the software worked as it should.

However, below is the requested information.

The unlocker folder is named: Drdonk_VMware_wkstn_golocker_423
The contents of the unlocker folder are:
backup\
iso\
linux\
templates\
windows\
CHANGELOG.md
LICENSE
README.md
TROUBLESHOOTING.md
The contents of the "backup" folder is a folder named 16.2.4.20089737, and its contents are files named:
vmwarebase.dll
vmwarebase.dll.sha256
vmware-vmx.exe
vmware-vmx.exe.sha256
vmware-vmx-debug.exe
vmware-vmx-debug.exe.sha256
vmware-vmx-stats.exe
vmware-vmx-stats.exe.sha256
